verticapy/sql/sql_magic.py

```diff
@@ -324,8 +324,3 @@ def sql(line, cell="", local_ns=None):
         # we load the previous configuration before returning the result.
         set_option("sql_on", sql_on)
         set_option("time_on", time_on)
-
-
-def load_ipython_extension(ipython):
```

Comment:
deleting a function here. is it not used anymore?

Reply:
I've added it elsewhere